bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/message-reactions/component.tsx

```
@@ -1,29 +1,102 @@
+
         return (
-          <Styled.EmojiWrapper highlighted={false}>
-            <em-emoji
-              emoji={emoji}
-              size={parseFloat(
-                window.getComputedStyle(document.documentElement).fontSize,
-              )}
-              native={emoji.native}
-            />
-          </Styled.EmojiWrapper>
+          <TooltipContainer title={label}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)